src/charts/stacked-area.js

```diff
@@ -588,6 +588,11 @@ define(function(require){
          * @return void
          */
         function drawGridLines(xTicks, yTicks) {
+
+            svg.select('.grid-lines-group')
+                .selectAll('line')
+                .remove();
+
```

Comment:
This makes more sense that what we have in the line chart, would you mind updating the line too?